setup.py

```diff
@@ -31,6 +31,8 @@
         'posthog',
         'ruamel.yaml',
         'alive-progress<=2.3.1'
+        'slack_sdk',
```

Comment:
Maybe let's limit these to their current major versions? for example slack_sdk < 4.0